XCUITests/PhotonActionSheetTest.swift

```diff
@@ -51,7 +52,7 @@ class PhotonActionSheetTest: BaseTestCase {
         let pageObjectButtonCenter = pageObjectButton.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0))
         pageObjectButtonCenter.press(forDuration: 1)
 
-        waitForExistence(app.cells["Copy"], timeout: 10)
+        waitForExistence(app.collectionViews.buttons["Copy"], timeout: 10)
```

Comment:
Can we add here the condition:
```
if #available(iOS 14.0, *) {
            waitForExistence(app.collectionViews.buttons["Copy"], timeout: 10)
        } else {
            waitForExistence(app.cells["Copy"], timeout: 10)
        }
```

We have not moved everything to iOS14 yet...so tests are going to fail in one or the other environment